src/components/Alert/Alert.tsx

```diff
@@ -3,13 +3,34 @@ Copyright (C) 2018 The Trustees of Indiana University
 SPDX-License-Identifier: BSD-3-Clause
 */
 import * as classNames from 'classnames';
+import * as PropTypes from "prop-types";
```

Comment:
We're using single quotes in imports elsewhere

```suggestion
import * as PropTypes from 'prop-types';
```